olive/passes/utils/whisper_prepost.py

```diff
@@ -14,6 +14,7 @@ def add_pre_post_processing_to_model(
     model_name: str,
     use_audio_decoder: bool = True,
     use_onnx_stft: bool = True,
+    skip_special_tokens: bool = True,
```

Comment:
I don't see the example use the parameter?

Reply:
https://github.com/microsoft/Olive/blob/dfb7f8e2038ec10fcefca39cd1b038a39b3ea74a/olive/passes/utils/whisper_prepost.py#L35

Reply:
Yes, the example doesn't use it. But since it's a configurable option like `use_onnx_stft`, I think exposing it is good.